lithops/standalone/proxy.py

```diff
@@ -81,16 +113,24 @@ def run():
     """
     global last_usage_time
     global jobs
+    global auto_dismantle
+    global soft_dismantle_timeout
+    global hard_dismantle_timeout
 
     message = flask.request.get_json(force=True, silent=True)
     if message and not isinstance(message, dict):
         return error()
 
     last_usage_time = time.time()
 
+    standalone_config = message['config']['standalone']
+    auto_dismantle = standalone_config['auto_dismantle']
+    soft_dismantle_timeout = standalone_config['soft_dismantle_timeout']
```

Comment:
instead of using global variables here, consider to use handler object fields

Reply:
When lithops uses a VM for the first time, it copies a config.json file in the VM that contains these values. This allows the proxy and the budget keeper to be automatically started when the VM is started at any time without receiving any request. The thing is that a user might decide to change those values at any point, so this ensures they are always updated